pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -246,6 +247,7 @@ func manageKubeAPIServerConfig(client coreclientv1.ConfigMapsGetter, recorder ev
 		"config.yaml",
 		specialMergeRules,
 		defaultConfig,
+		boundSATokenConfig,
```

Comment:
didn't we have the overrides yaml here as well?

Reply:
afaict the overrides yaml are only for bootstrap:

https://github.com/openshift/cluster-kube-apiserver-operator/tree/master/bindata/bootkube/config

I don't know why there are 2 files for bootstrap.  I do know that putting these options in the either of the bootstrap overrides will break bootstrapping because the bound token keypair is only created post-bootstrap.

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/bootkube/config/config-overrides.yaml

Reply:
I've posted a new PR to document the requirement to put post-bootstrap overrides in the renamed file: https://github.com/openshift/cluster-kube-apiserver-operator/pull/731